### PR TITLE
aiohttp: initial integration.

### PR DESCRIPTION
**Pull request by DavidKorczynski**
Integration of the aiohttp parser: https://github.com/aio-libs/aiohttp

Waiting for upstream.

### CODE REVIEW

**Comment by DavidKorczynski** [comment]
@inferno-chromium Could you give a comment on whether this project will be accepted? The maintainers are happy to integrate asks for some non-trivial updates, so would prefer to only do those updates if OSS-Fuzz would like the project:

Github specs for aiohttp 
- 10K stars on Github
- used by 61.9K on Github

Customers, based on aiohttp website:
https://docs.aiohttp.org/en/stable/external.html
- 20+ companies are powered by aiohttp: https://docs.aiohttp.org/en/stable/powered_by.html
- https://www.home-assistant.io/ (https://github.com/home-assistant/core)

For these reasons I can see that unexpected exceptions from aiohttp are important.

**Comment by inferno-chromium** [comment]
> @inferno-chromium Could you give a comment on whether this project will be accepted? The maintainers are happy to integrate asks for some non-trivial updates, so would prefer to only do those updates if OSS-Fuzz would like the project.
> 
> Github specs for aiohttp
> 
> * 10K stars on Github
> * used by 61.9K on Github
> 
> Customers, based on aiohttp website:
> https://docs.aiohttp.org/en/stable/external.html
> 
> * 20+ companies are powered by aiohttp: https://docs.aiohttp.org/en/stable/powered_by.html
> * https://www.home-assistant.io/ (https://github.com/home-assistant/core)
> 
> For these reasons I can see that unexpected exceptions from aiohttp are important.

Looks good based on https://docs.aiohttp.org/en/stable/third_party.html#approved-third-party-libraries as well

**Review by oliverchang — APPROVED** [review]

File: projects/aiohttp/fuzz_http_parser.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/python3
+
```
Comment (jonathanmetzman) [inline projects/aiohttp/fuzz_http_parser.py:2]: nit: delete this line.

File: projects/aiohttp/fuzz_http_parser.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import sys
+import atheris
+
+# aiohttp imports
```
Comment (jonathanmetzman) [inline projects/aiohttp/fuzz_http_parser.py:19]: nit: begin comments with capital letter and end with period.

**Review by jonathanmetzman — APPROVED** [review]
LGTM

**Pull request merged**